markdown/usage/nf_core_tutorial.md

```diff
@@ -288,63 +397,132 @@ nf-core lint /path/to/pipeline
 
-These test datasets are used by the automated continuous integration tests. The systems that run these tests are _extremely_ limited in the resources that they have available. Typically, the pipeline should be able to complete in around 10 minutes and use no more than 6-7 GB memory. To achieve this, input files and reference genomes need to be very tiny. If possible, a good approach can be to use PhiX or Yeast as a reference genome. Alternatively, a single small chromosome (or part of a chromosome) can be used. If you are struggling to get the tests to run, ask for help on Slack.
+The automated tests with GitHub Actions are configured in the `.github/worfklows` folder that is generated by the template.
+Each file Each file (`branch.yml`, `ci.yml` and `linting.yml`) defines several tests: branch protection, running the pipeline with the test data, linting the code with `nf-core lint`, linting the syntax of all Markdown documentation and linting the syntax of all YAML files.
 
-When writing `conf/test.config` remember to define all required parameters so that the pipeline will run with only `-profile test`. Note that remote URLs cannot be traversed like a regular file system - so glob file expansions such as `*.fa` will not work.
+#### `branch.yml`
 
-### Travis CI configuration
-The automated tests with Travis CI are configured in the `.travis.yml` file that is generated by the template. The `script` block defines three tests: linting the code with `nf-core lint`, linting the syntax of all Markdown documentation and running the pipeline with the test data.
+This is already set up and should not be touched.
+It will check that a PR is made on `dev` unless it's for a release or a patch.
 
-The `env` section sets the `NXF_VER` environment variable twice. This tells Travis to run the tests twice in parallel - once with the latest version of Nextflow (`NXF_VER=''`) and once with the minimum version supported by the pipeline. Do not edit this version number manually - it appears in multiple locations through the pipeline code, so it's better to use `nf-core bump-version --nextflow` instead.
+#### `linting.yml`
 
-The provided tests may be sufficient for your pipeline. However, if it is possible to run the pipeline with significantly different options (for example, different alignment tools), then it is good to test all of these. You can do this by adding additional commands in the `script` block.
+This is already set up and should not be touched.
+It will check:
+
+* That all Markdown documentation follow a proper syntax.
+(if you're using `atom` of `Visual Studio Code` or any similar editor, plugins are available to help you out: _e.g._ [markdownlint for atom](https://atom.io/packages/linter-markdownlint), [markdownlint for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint))
+* That all YAML file follow a proper syntax.
+* That the code follow the nf-core linting.
+
+#### `ci.yml`
+
```

Comment:
```suggestion

This file defines a GitHub Actions workflow that tests your pipeline with the small test dataset.
It should mostly work out of the box, but it may need some editing from you to customise it for your pipeline.

```